experiment/depth-first-traversal/simulation/js/treePractice.js

```diff
@@ -57,13 +57,13 @@ function check(d){
     if(tree_traversal.sequence_list[index]){
   
     document.getElementById("commentbox").style.display = "block";
-    let traversaltype = document.getElementById("trav").value;
+    let traversalType = document.getElementById("trav").value;
```

Comment:
traversalType should be declared as const (in place of let) as was mentioned in the previous review. This value is never going to be changed.
By default we declare each variable as const and change it to let when we encounter some error on changing its value.

Reply:
I agree but since the value of traversal type will change based on the user's choice of traversal, does it not become a variable ?

Reply:
Changes made , apologies for forgetting to change from let to const